community/STEERING-COMMITTEE.md

```diff
@@ -1,64 +1,156 @@
+  their organization shall nominate a new candidate.
+- If an organization is unable to seat a candidate, the KSC reserves the right
+  to reallocate the seat to another organization.
+- Changes to the number of seats, which company seats are allocated to, and
+  nominations to the committee are confirmed by majority vote of the committee
+  members.
+- In situations where the organization which holds a seat is no longer a viable
+  entity (e.g. merger, dissolution, bankruptcy) the KSC will make a decision on
+  how to reallocate that seat. Seats do not automatically transfer to any 
+  organization.
+
+## Clarifications
+
+_Why are seats company held?_
+
+The committee was created as the project was in its infancy, in order to tackle governance
+and overall project strategy. Because of the nature of the project and funding required,
+it was decided that strong corporate leadership was necessary for the project to ensure
+velocity.
+
+_What happens if someone leaves the company they represent?_
 
-<!-- TODO ## Committee Meeting -->
+In that case, the company will nominate a new representative for the seat. The nominee
+shall be confirmed by the standard process as called out above.
 
-## Committee Members
+_What happens if someone leaves the project or decides to leave the committee?_
 
-The members of the Steering Committee are shown below. Membership in the SC is
-determined by current level of contribution to the project. Contribution is
-periodically reviewed to ensure proper recognition.
+As above, the same would apply. The company who holds the seat will nominate a new
+representative to hold the seat. The nominee shall be confirmed by the standard process,
+called out above.
 
-| &nbsp;                                                   | Member         | Company | Profile                                  |
-| -------------------------------------------------------- | -------------- | ------- | ---------------------------------------- |
-| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google  | [@dewitt](https://github.com/dewitt)     |
-| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google  | [@mchmarny](https://github.com/mchmarny) |
-| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google  | [@isdal](https://github.com/isdal)       |
+_Is this steering model permanent?_
+
+The governance of Knative is an open, living document, and will continue to evolve as
+the community and project change. We expect over time we will adapt the way we run this
+committee, based on feedback from the community.
+
+_How are steering committee decisions reached?_
+
+The steering committee desires to always reach consensus. Decisions are made in
+meetings when a quorum of the members are present and may pass with majority
+of the committee supporting it. Quorum is considered reached when at least half
+of the members are present.
```

Comment:
And here is the difficult point. The quorum means that Google has full control of the project. Since 4 seats out of 7 are allocated to Google.

Reply:
This is true. These changes are mainly focused on taking a first step towards making the steering committee more open -- adding seats for additional non-Googlers, and ensuring that we have better transparency for when, how, and why decisions are made. It does not attempt to change the overall governance of the project, which has been in Google's control from the beginning of the project.

I think that's a reasonable question for the KSC to take up once these new rules are accepted.